### PR TITLE
Added in imageviz component, application and notebook

### PR DESCRIPTION
**Pull request by brechmos-stsci**
Created the imageviz component, application and jupyter notebook to show an example of the imageviz view.

I also added in a line to change the default `clim` from an upper bound of the maximum of the data, to an upper bound of the 99.5th percentile.  Some of the data has significant outliers and so the default view appeared to have no data.

Fixes https://github.com/spacetelescope/jdaviz/issues/28

### CODE REVIEW

**Comment by brechmos-stsci** [comment]
[As a reminder, this might need to be run in Google Chrome, for now as there is an intermittent bug in displaying image data using BQPlot in Firefox.]

File: examples/imageviz-example.ipynb [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    "## CubeViz Example"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "filename = '/home/craig/Documents/STScI/jdaviz/data/imviz/NIRISS_f150w_direct_dit1_nis_cal.fits'"
```
Comment (pllim) [inline examples/imageviz-example.ipynb]: Put this file somewhere on Box and let people know how to access it.

**Comment by brechmos-stsci** [comment]
I also was able to load a "big image is the CANDELS GOODS-South field. It is ~ 18k x 20k" that Cammi provided.  It actually loaded without an issue on my machine.  I was pleasantly surprised.  I think BQ plot is doing some decent downsampling for display that is probably helping quite significantly.

image

**Comment by hcferguson** [comment]
How fast does it load compared to ds9? Ginga?

File: examples/imageviz-example.ipynb (hunk 1 above)
Comment (brechmos-stsci) [reply examples/imageviz-example.ipynb]: I added in usage and moved the data to a Box folder accessible by all. The data loader was copied over from the cubeviz notebok.

**Comment by brechmos-stsci** [comment]
@hcferguson I would find it hard to compare as I have not used ds9 or Ginga.  Might be best to try the notebook out with local data and compare. 

Once the 1.4G data was downloaded on my machine, the display up in < 2 seconds.

**Comment by brechmos-stsci** [comment]
@javerbukh Understood. That is something we can work out later. This is just to confirm the glue-jupyter stuff will load a 2D dataset and display it.

**Pull request merged**